packages/@okta/vuepress-theme-prose/util/coveo/templates.js

```diff
@@ -0,0 +1,48 @@
+import config from "./config.js";
+
+const registerTemplates = () => {
+  Coveo.TemplateHelpers.registerTemplateHelper("sourceTag", function(source) {
+    let markup = "";
+    let commonSource = this.result.raw.commonoktasource[0];
```

Comment:
Is there any way to avoid using the `this` context here? Without understanding the Coveo framework it's impossible to know what context we're calling this function in. If the framework doesn't allow a better way then I suppose we can leave a comment instead.

Reply:
Not really sure how this part works, brought in as is from the Drupal site

Reply:
Not that much documentation around this on coveo's doc site and inspecting the params passed to the template, this seems to be the way to get the result data.